lib/susedistribution.pm

```diff
@@ -840,7 +839,7 @@ sub activate_console {
     }
     if (get_var('TUNNELED') && $name !~ /tunnel/) {
         die "Console '$console' activated in TUNNEL mode activated but tunnel(s) are not yet initialized, use the 'tunnel' console and call 'setup_ssh_tunnels' first" unless get_var('_SSH_TUNNELS_INITIALIZED');
-        (get_var('PUBLIC_CLOUD')) ? ssh_interactive_join() : $self->script_run('ssh -t sut', 0);
+        $self->script_run('ssh -t sut', 0);
         ensure_user($user) unless (get_var('PUBLIC_CLOUD'));
```

Comment:
I'm not sure about this line. I actually would expect that there is no difference for `PUBLIC_CLOUD` but 
`su - $user` give a login prompt again but the code afterwards isn't aware of it, or do I miss something?

Reply:
I'm not sure if `sut` is configured for publiccloud runs. But I share your concerns, this looks rather like a workaround than a necessity.

Reply:
yes we have, it is done here https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/publiccloud/ssh_interactive_init.pm#L47